packages/.gitignore

```diff
@@ -1,3 +1,4 @@
 *.patch.bz2
 *.src.rpm
 *.zip
+**/Cargo.lock
```

Comment:
Was this intentional? I thought we actually did want to track `Cargo.lock` files?

Reply:
This was in response to a preivous comment where my editor kept adding .lock files where not needed, sean suggested maybe we should setup a gitignore just for this level of Cargo.lock files

Reply:
> This was in response to a preivous comment where my editor kept adding .lock files where not needed, sean suggested maybe we should setup a gitignore just for this level of Cargo.lock files

That sounds like weird editor behavior - is it something you can change locally? I haven't seen spurious `Cargo.lock` files show up in past pull requests.